wp-api.js

```diff
@@ -10,8 +10,8 @@
 	};
 
 	/**
-	 * ECMAScript 5 shim, from MDN
-	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString
+	 * ECMAScript 5 shim, from MDN.
+	 * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString.
```

Comment:
Let me re-submit this PR and I will add `@link` tags for these URLs without a period at the end, as suggested in the core handbook.